phpunit.xml.dist

```diff
@@ -31,8 +31,6 @@ It's auto-generated by sonata-project/dev-kit package.
     <php>
         <ini name="precision" value="8" />
         <env name="SYMFONY_DEPRECATIONS_HELPER" value="max[self]=0" />
-        <env name="SYMFONY_PHPUNIT_REMOVE" value="symfony/yaml" />
```

Comment:
This was hiding the fact that we needed to declare phpspec prophecy as a dependency